docs/src/man/basics.md

```diff
@@ -1100,18 +1014,19 @@ julia> df1
 ```
 
 This is a non-copying operation. One can perform it only if `val` vector has the same length as number
-of rows of `df1` or as a special case if `df1` would not have any columns. 
+of rows of `df1` or as a special case if `df1` would not have any columns.
 
 ```jldoctest dataframe
 julia> df1.Age === val # no copy is performed
 true
 ```
 
-Below setting values of column `:Job` in rows `1:3` to values `[2, 4, 6]` *in-place* means write to an existing 
-column in a data frame.
+If select only some rows from a data frame the mutation is peformed in place, i.e.
```

Comment:
Not sure what this sentence means.

Reply:
fixed. Essentially means to say that if row selector is not `!` then in-place operation is performed